### PR TITLE
chore: bump otel metric version to 1.15.1

### PR DESCRIPTION
**Pull request by rogerogers**
chore: bump otel metric version to 1.15.1

升级otel metric版本到1.15.1

### CODE REVIEW

File: tracing/tracer_server.go [hunk 1]
```diff
@@ -18,12 +18,13 @@ import (
 	"context"
 	"time"
 
+	"go.opentelemetry.io/otel/metric"
```
Comment (YangruiEmma) [inline tracing/tracer_server.go:21]: please format the import

Comment (rogerogers) [reply tracing/tracer_server.go]: done

**Review by YangruiEmma — APPROVED** [review]

**Comment by rogerogers** [comment]
> is this pr ready to merge?

Yes, and we should release the two log packages before merging because the new version locks the go version to 1.19.

**Pull request merged**